includes/definitions/liebert.yaml

```diff
@@ -9,6 +9,7 @@ discovery:
     -
         sysObjectID:
             - .1.3.6.1.4.1.476.1.42
+            - .1.3.6.1.4.1.8072.3.2.10
```

Comment:
This is the standard linux sysObjectID... Looks like you might be able to use sysDescr instead at least.